services/brig/deb/etc/sv/brig/run

```diff
@@ -125,7 +125,8 @@ exec chpst -u $USER \
     --zauth-provider-token-timeout=${BRIG_ZAUTH_PROVIDER_TOKEN_TIMEOUT?'unset'} \
     --elasticsearch-url=${BRIG_ELASTICSEARCH_URL?'unset'} \
     --elasticsearch-user-index=${BRIG_ELASTICSEARCH_USER_INDEX?'unset'} \
-    --turn-servers=${BRIG_TURN_SERVERS?'unset'} \
+    --turn-servers-v1=${BRIG_TURN_SERVERS_V1?'unset'} \
+    --turn-servers-v2=${BRIG_TURN_SERVERS_V2?'unset'} \
```

Comment:
How about
```
    --turn-servers=${BRIG_TURN_SERVERS?'unset'} \
    --turn-servers-v2=${BRIG_TURN_SERVERS_V2?'unset'} \
```
(consistent with endpoint naming), to avoid having to lock the configuration commit to the code commit? (Otherwise, earlier config won't work with newer code and newer config won't work with earlier code - making any kind of rollback impossible)